packages/react-components/theme-designer/src/components/Swatch/Swatch.tsx

```diff
@@ -0,0 +1,196 @@
+  CalendarLtrRegular,
+} from '@fluentui/react-icons';
+
+export interface ContentProps {
+  className?: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)',
+    gridTemplateRows: 'repeat(3, auto)',
+    gap: tokens.spacingVerticalL,
+  },
+  avatar: {
+    gridArea: '1 / 1 / 2 / 2',
+    display: 'flex',
+    gap: tokens.spacingVerticalL,
+  },
+  tabList: {
+    gridArea: '2 / 1 / 3 / 2',
+  },
+  searchAndDropdownField: {
+    gridArea: '3 / 1 / 4 / 2/',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: tokens.spacingVerticalS,
+  },
+  textAndSwitch: {
+    gridArea: '1 / 2 / 2 / 3',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'start',
+    justifyContent: 'center',
+    justifySelf: 'center',
+    gap: tokens.spacingHorizontalL,
+  },
+  slider: {
+    gridArea: '2 / 2 / 3 / 3',
+    padding: tokens.spacingVerticalS,
+    width: '200px',
+    justifySelf: 'center',
+  },
+  switchButtons: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'start',
+    justifyContent: 'center',
+  },
+  checkboxAndRadioGroup: {
+    gridArea: '3 / 2 / 4 / 3',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+  checkboxGroup: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'start',
+    justifyContent: 'center',
+  },
+  radioGroup: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'start',
+    justifyContent: 'center',
+  },
+  descriptionField: {
+    gridArea: '1 / 3 / 2 / 4',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'start',
+    gap: tokens.spacingVerticalS,
+  },
+  icons: {
+    gridArea: '2 / 3 / 3 / 4',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto',
+    gridTemplateRows: 'auto auto',
+    rowGap: tokens.spacingVerticalS,
+    columnGap: tokens.spacingHorizontalS,
+    justifyContent: 'center',
+  },
+  link: {
+    gridArea: '3 / 3 / 4 / 4',
+    alignSelf: 'center',
+  },
+});
+
+const MeetNowIcon = bundleIcon(MeetNowFilled, MeetNowRegular);
+const CalendarLtrIcon = bundleIcon(CalendarLtrFilled, CalendarLtrRegular);
+/*
+  Note the state of the stickersheet is that we need consensus on the following post Build 2023:
+    1) Do we have all the correct components displayed and in the correct order?
+    2.a) Do we have all the states of each component displayed? i.e. missing deactive states?
+    2.b) Do was have all the variants of each component displayed? i.e. different sizes, colors, layouts etc?
+    3) Note that the spinner was removed since it was causing confusing with the loading state of the page
+*/
+
+export const Swatch: React.FC<ContentProps> = props => {
+  const stickerSheetStyles = useStyles();
+  const dropdownId = useId('dropdown-default');
+
+  return (
+    <div className={mergeClasses(stickerSheetStyles.root, props.className)}>
+      <div className={stickerSheetStyles.avatar}>
```

Comment:
is it possible to remove some of the wrapping divs and apply grid assignments directly to the default slot of the component like I did here?

https://stackblitz.com/edit/n1vqrywh?file=src%2Fexample.tsx

Reply:
I think we can even apply this to some of the wrapper components with children.